postman_tests/run_tests.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+import subprocess
+import os
+import json
+from lib import newman
+from lib import es
+import sys
+import traceback
+
+
+postman_tests_directory = 'postman_tests'
+entities_data_path = 'data/entities/'
+newman_data_path = 'data/newman_data.json'
+collection_data_path = 'data/ner_collection.json'
+es_data_path = 'data/elastic_search/'
+config_path = 'config'
+
+
+def get_newman_command():
+    if os.path.exists(f"{config_path}/dev.json"):
+        environment_file_path = f'{config_path}/dev.json'
+        return (
+            f'newman run {collection_data_path} -d {newman_data_path}'
+            f' -e {environment_file_path} -r cli,htmlextra --reporter-htmlextra-logs'
+        )
+    else:
+        environment_file_path = f'{config_path}/prod.json'
+        return (
+            f'newman run {collection_data_path} -d {newman_data_path}'
+            f' -e {environment_file_path}'
+        )
+
+
+# Switch to postman_tests if not already in that directory
+if(os.path.basename(os.getcwd()) != postman_tests_directory):
+    os.chdir(postman_tests_directory)
+
+
+if newman.check_if_data_valid(entities_data_path):
```

Comment:
This entire code block should not be in the global scope, should be written under some function which is called using
`__name__ == '__main__'` block

Reply:
Handle `else` case, tests should fail with a specific error if data is not valid, or instead of if-else, just let `newman.check_if_data_valid(entities_data_path)` return None but raise errors when data is invalid